buildconfig/pygame-stubs/rect.pyi

```diff
@@ -84,7 +84,7 @@ class Rect(object):
         ],
     ) -> None: ...
     @overload
-    def clamp(self, rect: Union[_CanBeRect, Rect]) -> Rect: ...
+    def clamp(self, rect: Union[CanBeRect, Rect]) -> Rect: ...
```

Comment:
`Union[CanBeRect, Rect]` can be replaced with `RectValue` here and many other places in this file. Sorry I forgot to fix it in my last PR but it'd be great if you could fix it in this one. Also there are more places where `CanBeRect` is used but `RectValue` can be used in place of that